packages/select/src/components/multi-select/_multi-select.scss

```diff
@@ -3,12 +3,25 @@
 
 @import "../../common/variables";
 
+$select-padding: $pt-grid-size * 0.5;
```

Comment:
Copied over from https://github.com/palantir/blueprint/blob/develop/packages/select/src/components/select/_select.scss

This moves the padding from Menu to the Popover-Content component. This is necessary as Menu is no longer the only rendered element in MultiSelect Popover(InputGroup can be present as well).